docs/tutorials/github/getting-started-github.md

```diff
@@ -0,0 +1,203 @@
+# Getting Started with GitHub Authentication
+
+This guide explains how to set up authentication with GitHub for Prowler. The documentation covers credential retrieval processes for each supported authentication method.
+
+## Prerequisites
+
+- GitHub account
+- Token creation permissions (organization-level access requires admin permissions)
+
+## Authentication Methods
+
+### 1. Personal Access Token (PAT)
+
+Personal Access Tokens provide the simplest GitHub authentication method and support individual user authentication or testing scenarios.
+
+#### How to Create a Personal Access Token
+
+1. **Navigate to GitHub Settings**
+   - Open [GitHub](https://github.com) and sign in
```

Comment:
To render like this (1 point with N sub-points) in mkdocs you need to add 1 extra indentation.